src/peripheralman/peripheralman.c

```diff
@@ -412,7 +412,15 @@ mraa_pman_i2c_stop_replace(mraa_i2c_context dev)
 static mraa_result_t
 mraa_pman_i2c_set_frequency_replace(mraa_i2c_context dev, mraa_i2c_mode_t mode)
 {
-    return MRAA_ERROR_FEATURE_NOT_SUPPORTED;
+    switch (mode) {
```

Comment:
Need a comment here that we default to the board's frequency but that when PeripheralManager supports the call... we'll fill in the action routines.